### PR TITLE
Add BOTWINGF722

### PR DESCRIPTION
**Pull request by haslinghuis**
5cd83077-434d-4beb-82fe-fd1ebf6704df

### CODE REVIEW

File: configs/BOTWINGF722/config.h [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+#define UART6_TX_PIN                        PC6
+#define UART6_RX_PIN                        PC7
+
+#define SERIALRX_UART                       SERIAL_PORT_USART2
+#define ESC_SENSOR_UART                     SERIAL_PORT_UART4
+#define GPS_UART                            SERIAL_PORT_USART6
+
+// TIMER && DMA
+#define TIMER_PIN_MAPPING                   TIMER_PIN_MAP(0, LED_STRIP_PIN, 1, 0) \
```
Comment (ot0tot) [inline configs/BOTWINGF722/config.h:129]: Conflicts with M4
```suggestion
#define TIMER_PIN_MAPPING                   TIMER_PIN_MAP(0, LED_STRIP_PIN, 2, 0) \
```

File: configs/BOTWINGF722/config.h [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+// TIMER && DMA
+#define TIMER_PIN_MAPPING                   TIMER_PIN_MAP(0, LED_STRIP_PIN, 2, 0) \
+                                            TIMER_PIN_MAP(1, MOTOR1_PIN,    1, 0) \
+                                            TIMER_PIN_MAP(2, MOTOR2_PIN,    1, 0) \
+                                            TIMER_PIN_MAP(3, MOTOR3_PIN,    1, 0) \
+                                            TIMER_PIN_MAP(4, MOTOR4_PIN,    1, 0) \
+                                            TIMER_PIN_MAP(5, MOTOR5_PIN,    2, 0) \
+                                            TIMER_PIN_MAP(6, MOTOR6_PIN,    2, 0) \
+                                            TIMER_PIN_MAP(7, MOTOR7_PIN,    1, 0) \
+                                            TIMER_PIN_MAP(8, MOTOR8_PIN,    1, 0)
```
Comment (ot0tot) [inline configs/BOTWINGF722/config.h:137]: Add RX_PPM support
```suggestion
#define UART2_RX_PIN                        PA3
#define RX_PPM_PIN                           PA3

#define UART4_RX_PIN                        PA1

#define UART6_TX_PIN                        PC6
#define UART6_RX_PIN                        PC7

#define SERIALRX_UART                       SERIAL_PORT_USART2
#define ESC_SENSOR_UART                     SERIAL_PORT_UART4
#define GPS_UART                            SERIAL_PORT_USART6

// TIMER && DMA
#define TIMER_PIN_MAPPING                   TIMER_PIN_MAP(0, LED_STRIP_PIN, 2, 0) \
                                            TIMER_PIN_MAP(1, MOTOR1_PIN,    1, 0) \
                                            TIMER_PIN_MAP(2, MOTOR2_PIN,    1, 0) \
                                            TIMER_PIN_MAP(3, MOTOR3_PIN,    1, 0) \
                                            TIMER_PIN_MAP(4, MOTOR4_PIN,    1, 0) \
                                            TIMER_PIN_MAP(5, MOTOR5_PIN,    2, 0) \
                                            TIMER_PIN_MAP(6, MOTOR6_PIN,    2, 0) \
                                            TIMER_PIN_MAP(7, MOTOR7_PIN,    1, 0) \
                                            TIMER_PIN_MAP(8, MOTOR8_PIN,    1, 0) \
                                            TIMER_PIN_MAP(9, RX_PPM_PIN ,    3, -1)
```

File: configs/BOTWINGF722/config.h [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+// MOTOR
+#define MOTOR1_PIN                          PB5
+#define MOTOR2_PIN                          PB4
+#define MOTOR3_PIN                          PB3
+#define MOTOR4_PIN                          PA15
+#define MOTOR5_PIN                          PB0
+#define MOTOR6_PIN                          PB1
+#define MOTOR7_PIN                          PB10
+#define MOTOR8_PIN                          PB11
```
Comment (ot0tot) [inline configs/BOTWINGF722/config.h:97]: Delete M5-8?

**Comment by nerdCopter** [comment]
i have compile errors with latest `47630a3` (compiling 4.5-maintenance)
```
In file included from ./src/main/platform.h:30,
                 from ./src/main/drivers/bus_i2c_timing.c:23:
./src/config/configs/BOTWINGF722/config.h:140:59: error: expected declaration specifiers or '...' before numeric constant
  140 |                                             TIMER_PIN_MAP(9, RX_PPM_PIN,    3, -1)
      |                                                           ^
%% (optimised) ./src/main/drivers/stm32/adc_stm32f7xx.c 
./src/config/configs/BOTWINGF722/config.h:119:45: error: unknown type name 'PA3'
  119 | #define RX_PPM_PIN                          PA3
      |                                             ^~~
./src/config/configs/BOTWINGF722/config.h:140:62: note: in expansion of macro 'RX_PPM_PIN'
  140 |                                             TIMER_PIN_MAP(9, RX_PPM_PIN,    3, -1)
      |                                                              ^~~~~~~~~~
./src/config/configs/BOTWINGF722/config.h:140:77: error: expected declaration specifiers or '...' before numeric constant
  140 |                                             TIMER_PIN_MAP(9, RX_PPM_PIN,    3, -1)
      |                                                                             ^
./src/config/configs/BOTWINGF722/config.h:140:80: error: expected declaration specifiers or '...' before '-' token
  140 |                                             TIMER_PIN_MAP(9, RX_PPM_PIN,    3, -1)
 
```

File: configs/BOTWINGF722/config.h (hunk 3 above)
Comment (haslinghuis) [reply configs/BOTWINGF722/config.h:97]: This PR was committed one day before the new policy got in place. 

@sugaarK @blckmn

**Review by ot0tot — APPROVED** [review]

**Review by nerdCopter — APPROVED** [review]
- awaiting MAG report, but approving now.

**Review by blckmn — APPROVED** [review]

**Pull request merged**